### PR TITLE
added c.py

### PR DESCRIPTION
**Pull request by ConorJohnsonMartin**

### CODE REVIEW

**Review by chrisatkinson16 — COMMENTED** [review]
conor

File: C.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def function_C():
+    print("C")
```
Comment (ghost) [inline C.py]: new line please

**Review by ghost — CHANGES_REQUESTED** [review]

File: C.py (hunk 1 above)
Comment (ConorJohnsonMartin) [reply C.py]: fixed it

**Review by chrisatkinson16 — COMMENTED** [review]
nice

**Pull request merged**